src/renderer.ts

```diff
@@ -123,17 +123,31 @@ const renderSlideObject = async (
         breakLine: true,
       });
     } else {
-      slide.addShape(shapeType, {
-        x,
-        y,
-        w,
-        h,
-        fill: backgroundColor,
-        line: {
-          size: style.borderWidth ?? undefined,
-          color: style.borderColor ?? undefined,
-        },
-      });
+      if(shapeType === 'line'){
+        slide.addShape(shapeType, {
+          x,
+          y,
+          w,
+          h,
+          fill: backgroundColor,
+          line: {
+            color: style.borderColor ?? undefined,
+            width: style.borderWidth ?? undefined
+          },
+        });
+      }else{
+        slide.addShape(shapeType, {
+          x,
+          y,
+          w,
+          h,
+          fill: backgroundColor,
+          line: {
+            size: style.borderWidth ?? undefined,
```

Comment:
If I'm reading pptxgenjs docs correctly, it would be OK to change this to be `width` as well. That way both line and other shape type blocks could be combined